README.md

```diff
@@ -31,12 +31,20 @@
 
 # Introduction
 
-The OSS Review Toolkit (ORT) aims to assist with the tasks that commonly need to be performed in the context of license
-compliance checks, especially for (but not limited to) Free and Open Source Software dependencies.
+The OSS Review Toolkit (ORT) is a FOSS policy automation and orchestration toolkit which you can use to manage your
+(open source) software dependencies in a strategic, safe and efficient manner.
 
-It does so by orchestrating a _highly customizable_ pipeline of tools that _abstract away_ the underlying services.
-These tools are implemented as libraries (for programmatic use) and exposed via a command line interface (for scripted
-use):
+You can use it to:
+
```

Comment:
I believe NOTICE files would be worth mentioning in the list of attribution documents below.

Reply:
FOSS attribution docs = NOTICE files - bullet points are no meant to describe every feature but as short & simple marketing messaging - in follow up commit I plan to add links to new sections that add the details.